File: api-service/Models/HousingSalePrice.cs
namespace ApiService.Models;

public class HousingSalePrice
{
    public int Id { get; set; }
    public int Year { get; set; }
    public string AreaName { get; set; } = string.Empty;
    public string AreaCode { get; set; } = string.Empty;
    public string AreaType { get; set; } = string.Empty;
    public decimal SumFloorAreaSold { get; set; }
    public decimal SumSalePrice { get; set; }
    public int NumberSales { get; set; }
    public decimal PricePerM2 { get; set; }
    public decimal? NumberBc { get; set; }
    public decimal? CostPerM2 { get; set; }
    public decimal? SumValueNew { get; set; }
    public decimal? Pcr { get; set; }
}

File: web-client/src/types/HousingSalePrice.ts
export interface HousingSalePrice {
  id: number
  year: number
  areaName: string
  areaCode: string
  areaType: string
  sumFloorAreaSold: number
  sumSalePrice: number
  numberSales: number
  pricePerM2: number
  numberBc: number | null
  costPerM2: number | null
  sumValueNew: number | null
  pcr: number | null
}
